cypress/integration/home.spec.ts

```diff
@@ -1,4 +1,67 @@
-it('should redirect to home page', () => {
-  cy.visit('/');
-  cy.location('pathname').should('eq', '/');
-});
+// it('should redirect to home page', () => {
+//   cy.visit('/');
+//   cy.location('pathname').should('eq', '/');
+// });
+
+// it('Should appear task name error when form is submitted empty', () => {
+//     cy.visit('/');
+//     cy.get('[data-testid="submit"]').click()
+//     cy.get('[data-testid="taskNameError"]').should('be.visible')
+// })
+
+// it('Should create task when form is submitted correctly', () => {
+//     cy.visit('/');
+//     cy.get('[data-testid="task"]').type('Go to market')
+//     cy.get('[data-testid="submit"]').click()
+//     cy.contains('Go to market')
+// });
+
+// it('Should remove task when delete button is clicked', () => {
+//     cy.visit('/');
+//     cy.contains('Remover').click()
+//     cy.contains('Go to shopping').should('not.exist')
+// });
+
+
+/**/
+
+it('Should raise an error when the input is empty ', () => {
+    cy.visit('/');
+    cy.get('[data-testid="taskTitle"]').click()
+    cy.contains('Adicionar tarefa').click()
```

Comment:
I suggest naming this selector inside a function:

```suggestion
    const getAddTodoButton = () => cy.get('[data-testid="add-todo-button"]');
    ...
    getAddTodoButton().click();
```

Reply:
Resolved on 695d4bfe9ad592adfb6a4dcab8192133c78af515